src/animation.cpp

```diff
@@ -0,0 +1,426 @@
+#include "game.h"
+#include "cata_tiles.h" // all animation functions will be pushed out to a cata_tiles function in some manner
```

Comment:
The unconditional inclusion of `cata_tiles.h` means we're also unconditionally requiring `SDL2/SDL.h`, even if we don't use it (as should be the case for lincurses, at the very least).